### PR TITLE
Updated the docs and ux

### PR DESCRIPTION
**Pull request by BShrivastav**
**NOTE:** updated for access_tags and ipv4_cidr_block

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000
https://jiracloud.swg.usma.ibm.com:8443/browse/UI-21204
Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Review by ujjwal-ibm — CHANGES_REQUESTED** [review]
consolidate both prs into one

File: website/docs/d/is_bare_metal_server.markdown [hunk 1]
```diff
@@ -111,7 +111,17 @@ In addition to all argument reference list, you can access the following attribu
 - `profile` - (String) The name for this bare metal server profile
 - `resource_group` - (String) resource group id of the bare metal server.
 - `resource_type` - (String) The type of resource referenced
-- `status` - (String) The status of the bare metal server [ **failed**, **pending**, **restarting**, **running**, **starting**, **stopped**, **stopping** ]
+- `status` - (String) The status of the bare metal server.
```
Comment (sumitkumartiwari) [inline website/docs/d/is_bare_metal_server.markdown:114]: Verify the below design with Harini if content team is fine, rest LGTM.

**Review by sumitkumartiwari — APPROVED** [review]
LGTM with comment, good to go upstream.

File: website/docs/d/is_bare_metal_server.markdown (hunk 1 above)
Comment (BShrivastav) [reply website/docs/d/is_bare_metal_server.markdown:115]: Discussed, and she agreed.

**Pull request closed without merging**